src/ThemedControls.cpp

```
@@ -249,16 +538,82 @@ void ThemedChoice::OnPaint(wxPaintEvent&) {
+  if (!m_open_popup) return;
+  m_open_popup->RequestClose();
+  m_open_popup = nullptr;
+}
+
+void ThemedChoice::MoveSelection(int delta) {
+  if (m_items.empty()) return;
+  const int i = std::clamp(m_selection + delta, 0,
+                           static_cast<int>(m_items.size()) - 1);
+  if (i == m_selection) return;
+  SetSelection(i);
   wxCommandEvent e(wxEVT_CHOICE, GetId());
```

Clang (14.0.6)

[warning] variable 'e' is not initialized

(cppcoreguidelines-init-variables)